common/src/main/java/io/homeassistant/companion/android/common/data/integration/Entity.kt

```diff
@@ -114,6 +115,16 @@ fun <T> Entity<T>.getCoverPosition(): EntityPosition? {
     }
 }
 
+fun <T> Entity<T>.supportsArmedAway(): Boolean {
```

Comment:
For consistency with other feature checks:

```suggestion
fun <T> Entity<T>.supportsAlarmControlPanelArmAway(): Boolean {
```